pages/your_role/researcher.md

```diff
@@ -11,26 +11,44 @@ training:
-
-
-## Focus
-
-* Write data management plans, also in the context of grant applications
-* Ensure compliance with institution policy, including legal and ethical aspects
-* Ensure proper data organisation and storage
-* Ensure secure sharing, reproducibility and preservation of data
-* Transmits the good practices in RDM to his group
-
-## Getting started
-
-* Check out the various steps of the [RDM life cycle](index), in particular the [planning](planning) stage
-* Identify and contact the data steward in your local organisation or your national contact in the [ELIXIR](https://elixir-europe.org/about-us/how-funded/eu-projects/converge/wp1/dm-coordinators) network
-* Start planning your project taking the [DMP](data_management_plan) into account
+## Introduction
+Your research data is a major output from your research project, it supports your research conclusions, and guides yourself and others towards future research. Therefore, managing the data well throughout the project, and sharing it, is a crucial aspect of research.
+
+## Data management responsibilities
+
+Data management includes actions throughout all stages of the research process. 
+
+In your role of researcher, you may need to:
+
+ * Write a data management plan;
+ * Estimate the costs of data management for a grant application;
+ * Ensure your data complies with ethical, policy and legal requirements;
+ * Store your data securely, in a resource that is sustained and accessible;
+ * Use metadata standards and documentation methods for your data;
+ * Get training about data management, including the use of data management tools and data archives.
+
+## Data management guidance
+
+### RDMkit pages
+
+ * [The RDM life cycle pages](https://rdmkit.elixir-europe.org/data_life_cycle) give an overview of how to approach data management tasks.
+ * [Data management plan](https://rdmkit.elixir-europe.org/data_management_plan) guides you through writing a data management plan.
+ * [Costs of data management](https://rdmkit.elixir-europe.org/costs_data_management) helps you budget for your project.
+ * [Compliance](https://rdmkit.elixir-europe.org/compliance_monitoring) helps you comply with the institution policy, including the legal and ethical aspects.
+ * [National resources](https://rdmkit.elixir-europe.org/national_resources) give country-specific guidance and resources.
+ * [Documentation and metadata](https://rdmkit.elixir-europe.org/metadata_management) helps you document data and choose the appropriate metadata standards.
+ * [Data publication](https://rdmkit.elixir-europe.org/data_publication) guides to find a data repository for your needs.
+
+### Other resources
```

Comment:
It seems that most of these links are ELIXIR resources. I would suggest to move these links in the metadata of the md page, so that they appear under "More information --> Link to other ELIXIR resources".

Reply:
I think we have to identify (1) what is the main content and purpose of the page, and (2) what is secondary content that goes in "related content" menus. I guess I'd say the purpose of the page is to gather the links to the most relevant parts of the site, and give them a context i.e. say why they might be useful. That is the main content of the page. If we put them all in "related content" menus then (a) I'm not sure if people will notice them and (b) there isn't much point having the page, since there won't be any main content. I think it's more of a portal page to the rest of the site than a page like a "Your domain" page or "Your task" page, where there are considerations and solutions to discuss. Not sure what others think.

Reply:
Agree with Martin, that's how we set up all new your role pages